api/contract_test.go

```diff
@@ -73,7 +73,7 @@ func TestCreateFiltersRequestValid(t *testing.T) {
 			r.Dimensions = r.Dimensions[:1]
 			Convey("When Valid() is called", func() {
 				err := r.Valid()
-				So(err, ShouldNotBeNil)
+				So(err, ShouldBeNil)
```

Comment:
This test no longer make much sense. Will it be better to update the test where a filter with 0 dimensions return an error, else we can remove the test

Reply:
I just removed it